packages/local-ganache-server/src/index.ts

```diff
@@ -24,7 +24,7 @@ export class LocalGanacheServer {
   server: any;
   networkContext: NetworkContextForTestSuite;
 
-  constructor(mnemonics: string[], initialBalance: string = "1000") {
+  constructor(extendedKeys: string[], initialBalance: string = "1000") {
```

Comment:
we should avoid the phrase "extended Key" because it doesn't specify private or public keys. I suggest this argument be named "xprvs"

Reply:
In some cases though, we can operate on either one

Reply:
yes, but (unless I'm mistaken) this function must take an extended private key, not an extended public key

Reply:
In the scope of this PR, that understanding is correct. The referenced PR changes this behavior though.

Reply:
you mean you can pass an array which contains both extended private keys and extended public keys as this argument? why would you do that?

Reply:
Nope.

It's explained in the PR's description.